packages/fe-fpm-writer/src/common/validate.ts

```diff
@@ -35,9 +35,10 @@ export function validateBasePath(basePath: string, fs?: Editor): boolean {
         throw new Error(`Invalid project folder. Cannot find required file ${manifestPath}`);
     } else {
         const manifest = fs.readJSON(manifestPath) as any;
-        if ((manifest['sap.ui5']?.dependencies?.libs?.['sap.fe.templates'] !== undefined) === false) {
+        const ui5Dependencies = manifest['sap.ui5']?.dependencies?.libs;
+        if (!(ui5Dependencies && (ui5Dependencies['sap.fe.core'] || ui5Dependencies['sap.fe.templates']))) {
```

Comment:
`sap.fe.macros` works as well

Reply:
tests are failing with `sap.fe.macros` for fpm, any other suggestions?

Reply:
fpm apps seem to need these added to the ui5-local.yaml. 
``
    - name: sap.fe.macros
    - name: sap.fe.templates
```
test app fails to load from `/packages/fiori-elements-writer/test/test-output/fpmTemplates/with-js`

Reply:
Suggestions to add them specifically for FPM? Currently preloading `sap.ushell` for ui5-local.yaml, extending this to include `sap.fe.templates` and `sap.fe.macros` could mean unnecessary code for other apps ui5-local.yaml ?

```
    // preload ui5-local.yaml with 'sap.ushell'
    const ui5Libs = ui5App.ui5.ui5Libs as string[];
    if (!ui5Libs.includes('sap.ushell')) {
        ui5Libs.push('sap.ushell');
    }
 ```
 
Looks to me that this is the point where the yaml file is written, and nothing to distinguish apps based on ui5-local.yaml ? Maybe can read manifest to determine template type, if fpm preload `templates` and `macros`?